src/containerapp/azext_containerapp/tests/latest/httprouteconfig1.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
```

Comment:
About these 2 files, seems they can be added and removed by code, can you refer `test_containerapp_preview_create_with_yaml` ?

For example:
```

containerapp_file_name = f"{self._testMethodName}_containerapp.yml"

write_test_file(containerapp_file_name, containerapp_yaml_text)
...do test
clean_up_test_file(containerapp_file_name)

```